job-runner.toml

```diff
@@ -0,0 +1,37 @@
+# Install to ~/.config/cockpit-dev/job-runner.toml
+
+[container]
+command = 'podman'
+args = []
+run-args = ['--device=/dev/kvm']
+default-image = 'quay.io/cockpit/tasks:latest'
+
+[container.secrets]
+# see podman-secret(1)
+# github-token = '--secret=github-token'
+
+[logs]
+driver='local'   # 's3' or 'local'
+
+[logs.s3]
+# hint: podman run --rm --net=host quay.io/minio/minio server /var
```

Comment:
Ya.  This is done for exceptionally evil reasons: `/var` was chosen specifically because it already contains a world-writable directory called `tmp`, which `minio` is then prepared to serve as a bucket called `tmp`.

This is the only way I could find to get a semi-functional config out of the box without needing to use `mc`.  It's also not completely functional, though: you can't read anything without a token.  In general I strongly prefer either a real-world S3 bucket ([play.min.io](https://play.min.io:9443/) is interesting if you don't otherwise have something you can use) or the default `local` backend along with `python -m http.server`.

Reply:
Hah, thansk for the explanation :nerd_face: